typings/index.d.ts

```diff
@@ -2292,6 +2293,8 @@ export class GuildChannelManager extends CachedManager<
   ): Promise<
     Collection<Snowflake, TextChannel | VoiceChannel | CategoryChannel | NewsChannel | StoreChannel | StageChannel>
   >;
+  public fetchActiveThreads(cache?: boolean): Promise<FetchedThreads>;
+  public fetchActiveThreads(cache: boolean, returnRaw: true): Promise<APIThreadList>;
```

Comment:
`cache` has no effect if `returnRaw` is true - perhaps change to a single object parameter and omit cache on this overload?